apps/OboeTester/docs/AutomatedTesting.md

```diff
@@ -21,8 +21,8 @@ Before running the app from an Intent, it should be launched manually and a Roun
 The app can be started by sending a Start comment to the OboeTester class.
 The app will run and the results will be written to a file.
 
-    adb shell am start -n com.google.sample.oboe.manualtest/.MainActivity {parameters}
-    
+    adb shell am start -n com.mobileer.manualtest/.MainActivity {parameters}
```

Comment:
These should be com.mobileer.oboetester.
I have a fix but GitHub won't let me push it. So I will fix it manually.